templates/zerver/features.html

```diff
@@ -1,5 +1,8 @@
 {% extends "zerver/portico.html" %}
 
+{% set OPEN_GRAPH_TITLE = 'page title' %}
+{% set OPEN_GRAPH_DESCRIPTION = 'page description' %}
```

Comment:
I think this approach is cleaner in any case, since we don't duplicate as much code, so let's run with it!

@rishig this might be all you need to draft these for /features and a few friends, since the remaining work will be entirely writing content...